config/shoryuken.yml

```diff
@@ -18,3 +18,8 @@ queues:
   - ["<%= prefix %>_announce_feeder_story_delete", 1]
   - ["<%= prefix %>_announce_feeder_story_publish", 1]
   - ["<%= prefix %>_announce_feeder_story_unpublish", 1]
+groups:
+  single_threaded:
```

Comment:
I've never heard of / tried this!  But I'm guessing it worked locally for you.

I do wonder how this will scale.  How long do these typically take?  And how many shows could we theoretically handle before the lag gets out of control?  (Good problems to solve later - but I'm just curious).

Reply:
I was able to spin up a job queue and run a job.

In the typical case where we are publishing 'one more' episode, the time is roughly a minute or two.  There is a strong assumption in here that we wait for the episode to finish processing completely.  In fact you cannot mark an episode from DRAFTING to PUBLISHED in the api with the media settling down completely.